src/js/angular/clustermanagement/templates/cluster-configuration/multi-region.html

```diff
@@ -1 +1,146 @@
+                            <div class="actions">
+                                <button ng-click="cancel()"
+                                        class="btn btn-link cancel-tag-add-btn secondary"
+                                        gdb-tooltip="{{'cluster_management.cluster_configuration_multi_region.cancel_tooltip' | translate}}">
+                                    <i class="fa-regular fa-xmark"></i>
+                                </button>
+                                <button ng-click="createTag(asciiInput)"
+                                        class="btn btn-link save-rule-btn"
+                                        ng-disabled="!tagForm.$valid"
+                                        gdb-tooltip="{{'cluster_management.cluster_configuration_multi_region.create_tag' | translate}}">
+                                    <i class="fa-regular fa-check"></i>
+                                </button>
+                            </div>
+                        </td>
+                    </tr>
+
+                    <tr ng-repeat-start="tag in topology.primaryTags track by $index" ng-if="0"></tr>
+                    <tr class="preview-tag-row">
+                        <td class="index-cell"><span>{{$index + 1}}</span></td>
+                        <td class="tag-name-cell">
+                            {{tag[0]}}
+                        </td>
+                        <td class="tag-index-cell">
+                            <span>{{tag[1]}}</span>
+                        </td>
+                        <td class="actions-cell">
+                            <div class="actions">
+                                <button ng-click="deleteTag(tag[0])"
+                                        ng-disabled="addingTag"
+                                        class="btn btn-link delete-tag-btn secondary"
+                                        gdb-tooltip="{{ 'cluster_management.cluster_configuration_multi_region.delete_tag_tooltip' | translate }}"
+                                        title-class="delete-tag-tooltip">
+                                    <i class="fa-regular fa-trash-can"></i>
+                                </button>
+                            </div>
+                        </td>
+                    </tr>
+                    <tr ng-repeat-end ng-if="0"></tr>
+                    </tbody>
+                </table>
+                <div ng-show="tagForm.asciiInput.$error.asciiValidator && !tagForm.asciiInput.$pristine">
+                    <span class="error-message">{{ 'cluster_management.cluster_configuration_multi_region.error.ascii' | translate }}</span>
+                </div>
+                <div ng-show="tagForm.asciiInput.$error.lengthValidator && !tagForm.asciiInput.$pristine">
+                    <span class="error-message">{{ 'cluster_management.cluster_configuration_multi_region.error.length' | translate: {minLen: '3', maxLen: '255'} }}</span>
```

Comment:
I'd be good if you added a small model in the controller for this `{minLen: '3', maxLen: '255'}` and reused it for the ascii input as well

Reply:
done